### PR TITLE
Remove the primary exporter/receiver components, update references and documentation

### PR DESCRIPTION
**Pull request by jmacd**
The Exporter and Receiver components are now included in the Collector-Contrib release (components in v0.102.0) and the released Docker images (as of v0.105.0).

### CODE REVIEW

File: collector/README.md [hunk 1]
```diff
@@ -11,20 +11,8 @@ The primary components are:
...
-
-After you have built a collector using one of the documented methods,
-see the [examples][EXAMPLES].
-
-[We would prefer to include these components in the OpenTelemetry
-Contrib Collector, because it is an officially maintained artifact.
-At this time, however, these components are new and the migration
-process will take some time to complete.][CONTRIBUTION]
+The exporter and receiver components are included in the official
+OpenTelemetry Collector-Contrib release images since v0.105.0.
```
Comment (lquerel) [inline collector/README.md:15]: I suggest adding a link to the configuration/installation procedure or to the OpenTelemetry contribs collector builder.

**Review by lquerel — APPROVED** [review]
I made a minor suggestion. 

This is seriously cool to finally see the exporter/receiver directly included in the OTEL collector contrib repo. Thanks @jmacd  and @moh-osman3 !

File: collector/README.md (hunk 1 above)
Comment (jmacd) [reply collector/README.md:15]: I also noticed some broken links remaining here, will revise.

Comment (jmacd) [reply collector/README.md:15]: I will do this in a new PR, since this one is fragile.

**Pull request merged**